src/data/markdown/translated-guides/en/05 Test Types/03 Stress testing.md

```diff
@@ -46,153 +46,179 @@ We recommend running a stress test in a UAT or staging environment.
-Lets start with an example:
+For stress tests, `ramping-arrival-rate` makes a suitable executor, as you can set raw throughput as a value in iterations per second.
+
+Remember, the point of this test is to gradually push your APIs beyond its breaking point,
+for example:
 
 <CodeGroup labels={["API stress test k6 example"]} lineNumbers={[true]} heightTogglers={[true]}>
 
 ```javascript
-import http from 'k6/http';
-import { sleep } from 'k6';
+import http from "k6/http";
+import { sleep } from "k6";
 
 export const options = {
-  stages: [
-    { duration: '2m', target: 100 }, // below normal load
-    { duration: '5m', target: 100 },
-    { duration: '2m', target: 200 }, // normal load
-    { duration: '5m', target: 200 },
-    { duration: '2m', target: 300 }, // around the breaking point
-    { duration: '5m', target: 300 },
-    { duration: '2m', target: 400 }, // beyond the breaking point
-    { duration: '5m', target: 400 },
-    { duration: '10m', target: 0 }, // scale down. Recovery stage.
-  ],
+  scenarios: {
+    stress: {
+      executor: "ramping-arrival-rate",
+      preAllocatedVUs: 500,
+      timeUnit: "1s",
+      stages: [
+        { duration: "2m", target: 10 }, // below normal load
+        { duration: "5m", target: 10 },
+        { duration: "2m", target: 20 }, // normal load
+        { duration: "5m", target: 20 },
+        { duration: "2m", target: 30 }, // around the breaking point
+        { duration: "5m", target: 30 },
+        { duration: "2m", target: 40 }, // beyond the breaking point
+        { duration: "5m", target: 40 },
+        { duration: "10m", target: 0 }, // scale down. Recovery stage.
+      ],
+    },
+  },
 };
 
 export default function () {
-  const BASE_URL = 'https://test-api.k6.io'; // make sure this is not production
-
+  const BASE_URL = "https://test-api.k6.io"; // make sure this is not production
   const responses = http.batch([
-    ['GET', `${BASE_URL}/public/crocodiles/1/`, null, { tags: { name: 'PublicCrocs' } }],
-    ['GET', `${BASE_URL}/public/crocodiles/2/`, null, { tags: { name: 'PublicCrocs' } }],
-    ['GET', `${BASE_URL}/public/crocodiles/3/`, null, { tags: { name: 'PublicCrocs' } }],
-    ['GET', `${BASE_URL}/public/crocodiles/4/`, null, { tags: { name: 'PublicCrocs' } }],
+    ["GET", `${BASE_URL}/public/crocodiles/1/`],
+    ["GET", `${BASE_URL}/public/crocodiles/2/`],
+    ["GET", `${BASE_URL}/public/crocodiles/3/`],
+    ["GET", `${BASE_URL}/public/crocodiles/4/`],
   ]);
-
-  sleep(1);
 }
+
 ```
 
 </CodeGroup>
 
 The VU chart of a stress test should look similar to this:
-![Virtual user chart of an API stress test](./images/stress-test.png)
-
-This configuration increases the load by 100 users every 2 minutes and stays at this level for
+![Virtual user chart of an API stress test](./images/stress-test.png "This configuration increases the load by 100 users every 2 minutes and stays at this level for
```

Comment:
The comment is accurate for hte picture but not the test above. I am not certain there needs to be a picture *with* VUs here  - the important part is that the required target is hit. Not certain if there is a good graph for that :shrug: